packages/manager/apps/dedicated/client/app/ip/components/list/list.html

```diff
@@ -81,6 +81,10 @@
                         class="oui-popover-button"
                         data-oui-popover="{{:: 'ip_table_header_ip_address_help' | translate }}"
                         data-oui-popover-placement="top"
+                        data-oui-popover-scope="ipBlock.ipBlock"
+                        data-oui-popover-open="ipBlock.ipBlock"
+                        data-ng-mouseover="ipBlock.ipBlock = true"
+                        data-ng-mouseleave="ipBlock.ipBlock = false"
```

Comment:
**nitpick:** Since `ipBlock` is used as an iterator on `tbody` it could lead to confusion of why the variables are set here. So better to rename the popover flags.
```suggestion
                        data-oui-popover-scope="displayIpBlockPopover"
                        data-oui-popover-open="displayIpBlockPopover"
                        data-ng-mouseover="displayIpBlockPopover = true"
                        data-ng-mouseleave="displayIpBlockPopover = false"
```